app/controllers/feedbacks_controller.rb

```diff
@@ -177,4 +159,18 @@ def find_feedback_for(user)
       end
     end
   end
+
+  def execute_button_action(format)
+    if params[:submit_final_button]
+      @feedback.submit_final
+      flash[:success] = 'Feedback was submitted.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.